plugins/module_utils/mh/module_helper.py

```diff
@@ -26,23 +22,10 @@ class ModuleHelper(DeprecateAttrsMixin, ModuleHelperBase):
     diff_params = ()
     change_params = ()
     facts_params = ()
-    use_old_vardict = True      # remove in 11.0.0
-    mute_vardict_deprecation = False
 
     def __init__(self, module=None):
-        if self.use_old_vardict:  # remove first half of the if in 11.0.0
-            self.vars = _OldVarDict()
-            super(ModuleHelper, self).__init__(module)
-            if not self.mute_vardict_deprecation:
-                self.module.deprecate(
-                    "This class is using the old VarDict from ModuleHelper, which is deprecated. "
-                    "Set the class variable use_old_vardict to False and make the necessary adjustments."
-                    "The old VarDict class will be removed in community.general 11.0.0",
-                    version="11.0.0", collection_name="community.general"
-                )
-        else:
-            self.vars = _NewVarDict()
-            super(ModuleHelper, self).__init__(module)
+        self.vars = VarDict()
+        super(ModuleHelper, self).__init__(module)
 
         for name, value in self.module.params.items():
```

Comment:
Cosmetic change for readability.
```suggestion
        super(ModuleHelper, self).__init__(module)

        self.vars = VarDict()
        for name, value in self.module.params.items():
```